src/frontend/Sider/Behandling/Brev/useVerdierForBrev.ts

```diff
@@ -0,0 +1,30 @@
+import { BeregningsresultatTilsynBarn } from '../../../typer/vedtak';
+import { formaterNullableIsoDato } from '../../../utils/dato';
+
+export type VariabelStore = { [variabelId: string]: string };
+
+export type Brevverdier = {
+    variabelStore: VariabelStore;
+};
+
+export const useVerdierForBrev = (
+    beregningsresultat?: BeregningsresultatTilsynBarn
+): Brevverdier => {
+    const variabelIdVedtakFraDato = 'f194e1fa-ba7a-4338-96fb-1d2b1a586dd3';
+    const VariabelIdVedtakTilDato = 'e67f3fe5-a318-447b-97c4-1495c132b61e';
+
+    const stønadsperioder = beregningsresultat?.perioder[0].grunnlag.stønadsperioderGrunnlag;
+
+    const fraDato = formaterNullableIsoDato(stønadsperioder?.[0].stønadsperiode.fom) || '';
+    const tilDato =
+        formaterNullableIsoDato(
+            stønadsperioder?.[stønadsperioder?.length - 1].stønadsperiode.tom
```

Comment:
Er tanken at denne skal finne maks tom per "periode"? `beregningsresultat?.perioder[0]` brukes her 👀 
Eller skal man egentlige finne maks for alle perioder?

Reply:
Kan ev bruke `min(perioder.map(stønadsperiode.fom))` og `max(perioder.map(stønadsperiode.tom))`

Reply:
Likte dette forslaget^